go/vt/vttablet/endtoend/transaction_test.go

```diff
@@ -766,3 +773,66 @@ func TestUnresolvedTransactions(t *testing.T) {
 	}}
 	utils.MustMatch(t, want, transactions)
 }
+
+// TestUnresolvedTransactions tests the UnresolvedTransactions API.
+func TestUnresolvedTransactionsOrdering(t *testing.T) {
+	client := framework.NewClient()
+
+	participants1 := []*querypb.Target{
+		{Keyspace: "ks1", Shard: "c0-"},
+		{Keyspace: "ks1", Shard: "80-c0"},
+	}
+	participants2 := []*querypb.Target{
+		{Keyspace: "ks1", Shard: "-40"},
+		{Keyspace: "ks1", Shard: "80-c0"},
+	}
+	participants3 := []*querypb.Target{
+		{Keyspace: "ks1", Shard: "c0-"},
+		{Keyspace: "ks1", Shard: "-40"},
+	}
+	// prepare state
+	err := client.CreateTransaction("dtid01", participants1)
+	require.NoError(t, err)
+	defer client.ConcludeTransaction("dtid01")
```

Comment:
why are we doing `ConcludeTransaction` here? we are not testing this, right?

Reply:
This is needed to clear out the pending transaction before the next test.